### DISCUSSION TITLE
Issue with Running PLOG Mechanism in PeleC

### DISCUSSION THREAD

**Discussion by CAIROCharlie** [AMReX-Combustion/PeleC#879] [Q&A]
Dear all,

I used the method from this link: https://github.com/AMReX-Combustion/PelePhysics/pull/549 to convert the PLOG mechanism and set the input to 1 atm. However, after running the case, the reaction started but this issue occurred shortly after:
```
[ERROR][rank 0][/data/PeleC/Submodules/PelePhysics/Submodules/sundials/src/cvode/cvode.c:3683][cvHandleFailure] At t = 0 and h = 3.8938e-18, the corrector convergence test failed repeatedly or with |h| = hmin.
[ERROR][rank 0][/data/PeleC/Submodules/PelePhysics/Submodules/sundials/src/cvode/cvode.c:3683][cvHandleFailure] At t = 0 and h = 3.8938e-18, the corrector convergence test failed repeatedly or with |h| = hmin.
[ERROR][rank 0][/data/PeleC/Submodules/PelePhysics/Submodules/sundials/src/cvode/cvode.c:3683][cvHandleFailure] At t = 0 and h = 3.8938e-18, the corrector convergence test failed repeatedly or with |h| = hmin.
...
 Signalling a stop because NaNs detected in the Solution
amrex::Abort::6::Simulation stopped early !!!
amrex::Abort::7::Simulation stopped early !!!
amrex::Abort::13::Simulation stopped early !!!
amrex::Abort::15::Simulation stopped early !!!
amrex::Abort::17::Simulation stopped early !!!
amrex::Abort::22::Simulation stopped early !!!
amrex::amrex::Abort::25::Simulation stopped early !!!
```
Could you please help me with this?
Thank you very much!

**Comment by baperry2** [comment]
What case are you running? I'd recommend testing the mechanism you generated for a simple case (say `ReactEval` in PelePhysics) to verify that the mechanism is working as expected before moving to something complicated. 

Another recommendation is to visualize the state at the step immediately before where it fails and check for anything unusual (out fo bounds mass fractions or temperature).

**Reply by CAIROCharlie** [reply]
Thank you very much for your help,  I am trying the options you mentioned.
